cdap-security-spi/src/main/java/io/cdap/cdap/security/spi/authentication/SecurityRequestContext.java

```diff
@@ -97,5 +101,19 @@ public static void reset() {
     userId.remove();
     userIP.remove();
     userCredential.remove();
+    auditLogContextQueue.remove();
+  }
+
+  public static void enqueueAuditLogContext(AuditLogContext auditLog) {
+    if (auditLogContextQueue == null) {
```

Comment:
`auditLogContextQueue` will never be null. `auditLogContextQueue.get()` will be null